presentation/src/main/java/com/anytypeio/anytype/presentation/search/GlobalSearchViewModel.kt

```diff
@@ -130,6 +248,37 @@ class GlobalSearchViewModel(
             ) as T
         }
     }
+
+    private sealed class Mode {
```

Comment:
add a description pls of what these modes are

Reply:
done